include/mfmg/dealii_adapters.hpp

```diff
@@ -36,7 +36,7 @@ class DealIIMeshEvaluator
   using vector_type = VectorType;
   using value_type = typename VectorType::value_type;
   using operator_type = Operator<vector_type>;
-  using global_operator_type = DealIITrilinosMatrixOperator<vector_type>;
+  using global_operator_type = DealIITrilinosMatrixfreeOperator<vector_type>;
```

Comment:
This means that we cannot use the matrix-based without recompiling the code and thus, we cannot test the matrix-based code. Can we instead derive `DealIIMatrixFreeMeshEvaluator` from `DealIIMeshEvaluator`? We will need to add a specialization of `Adapter` though. I think that we should have used inheritance instead of template specialization for `Adapted` but that's for another PR.